ic-os/components/hostos-scripts/monitoring/monitor-power.sh

```diff
@@ -38,7 +38,7 @@ write_line() {
     #    Minimum during sampling period:                132 Watts
     #    Maximum during sampling period:                384 Watts
     #    Average power reading over sample period:      204 Watts
-    #    IPMI timestamp:                           03/06/2025 07:40:35 PM UTC    Sampling period:                          00000001 Seconds.
+    #    IPMI timestamp:                           03/06/2025 07:40:35 UTC    Sampling period:                          00000001 Seconds.
     #    Power reading state is:                   activated
 
     ipmi_output="$(ipmitool dcmi power reading 2>/dev/null)"
```

Comment:
I wonder if this is a term or locale thing, could there be a better way to use this as "plumbing"?

Maybe:
> -c
> Present output in CSV (comma separated variable) format. This is not available with all commands.

Reply:
Just tried it on the baremetal machine, but it doesn't work for that command 😢

Reply:
I guess we could make the call directly with `ipmitool raw` by putting [this](https://github.com/ipmitool/ipmitool/blob/be11d948f89b10be094e28d8a0a5e8fb532c7b60/lib/ipmi_dcmi.c#L1403-L1419), [here](https://github.com/ipmitool/ipmitool/blob/be11d948f89b10be094e28d8a0a5e8fb532c7b60/lib/ipmi_raw.c#L313), and interpreting the response directly as [this](https://github.com/ipmitool/ipmitool/blob/be11d948f89b10be094e28d8a0a5e8fb532c7b60/include/ipmitool/ipmi_dcmi.h#L119-L128).

but that's... gross.

---

I think [this](https://github.com/ipmitool/ipmitool/blob/be11d948f89b10be094e28d8a0a5e8fb532c7b60/lib/ipmi_time.c#L218) format (`%x %X %Z`) changes with locale, which makes the difference.